go/ephemeral/user_ek_box_storage.go

```diff
@@ -112,6 +112,10 @@ func (s *UserEKBoxStorage) fetchAndPut(ctx context.Context, generation keybase1.
 		return userEK, err
 	}
 
+	if result.Result == nil {
+		return userEK, fmt.Errorf("server didn't return a box for userEK generation %d", generation)
+	}
```

Comment:
I triggered this condition by writing a bug. Better to check for nil explicitly than to proceed with zero'd data, I think.

Reply:
yeah i like that better, do we need to make similar changes for the `teamEKBoxRepsonse` case? (or any other server responses?h

Reply:
Yeah if we expect the server to be null sometimes, we should be explicitly checking for it.

Reply:
can you go through and double check any other places we may have forgotten this?

Reply:
I've added handling for the team box case, and it looks like we were already handling this sort of thing in `fetchUserEKStatement`.